src/main/java/frc/robot/constants/ArmConstants.java

```diff
@@ -58,5 +58,23 @@ public enum ArmPositions {
             this.angle = angle;
             this.sensorUnits = angle / GEAR_RATIO * CANCoderConstants.COUNTS_PER_DEG;
         }
+
+        public String toString() {
+            switch (this) {
+                case UP:
+                    return "UP";
+                case DOWN:
+                    return "DOWN";
+                case CONE_FLIP:
+                    return "CONE_FLIP";
+                case SHOOT:
+                    return "SHOOT";
+                case PORTAL:
+                    return "PORTAL";
+                case HANDOFF:
+                    return "HANDOFF";
+            }
+            return "";
+        }
```

Comment:
You don't need to override the built in toString of the enum. You can use enum.toString() or enum.name().